extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -133,18 +133,24 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 	private readonly _requiredPackagesSet: Set<string>;
 
 	private readonly _runningOnSAW: boolean;
+	private readonly _tsgops: boolean;
 
 	constructor(extensionPath: string, outputChannel: vscode.OutputChannel) {
 		this.extensionPath = extensionPath;
 		this.outputChannel = outputChannel;
 
 		this._runningOnSAW = vscode.env.appName.toLowerCase().indexOf('saw') > 0;
+		this._tsgops = vscode.env.appName.toLowerCase().indexOf('tsgops') > 0;
 		void vscode.commands.executeCommand(constants.BuiltInCommands.SetContext, 'notebook:runningOnSAW', this._runningOnSAW);
 
 		if (this._runningOnSAW) {
 			this._pythonInstallationPath = `${vscode.env.appRoot}\\ads-python`;
 			this._usingExistingPython = true;
-		} else {
+		} else if (this._tsgops) {
+			this._pythonInstallationPath = `/usr`;
```

Comment:
As @alanrenmsft pointed out - is this guaranteed to be accurate? Will this never be running on a windows machine?

Reply:
This is only planned to be used for adsweb which is based on linux